independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/core/startup/RuntimeDeploymentManager.java

```diff
@@ -171,28 +171,53 @@ public void accept(String method, RequestMapper<RuntimeResource> mapper) {
         //we use this map to merge them
         Map<MappersKey, Map<String, TreeMap<URITemplate, List<RequestMapper.RequestPath<RuntimeResource>>>>> mappers = new TreeMap<>();
 
-        for (int i = 0; i < resourceClasses.size(); i++) {
-            ResourceClass clazz = resourceClasses.get(i);
-            if ((clazz.getIsDisabled() != null) && clazz.getIsDisabled().get()) {
-                continue;
-            }
-            URITemplate classTemplate = new URITemplate(clazz.getPath(), true);
+        List<List<ResourceClass>> rcMap = List.of(locatableResourceClasses, resourceClasses);
```

Comment:
This logic is what picks up the static methods on locatable subresources. Locally it seems to work, both when I run the test and if I create an example app. I'm not sure what could be different about CI that causes it to fail.